fe/fe-core/src/main/java/com/starrocks/qe/ConnectProcessor.java

```diff
@@ -226,7 +226,6 @@ private void addFinishedQueryDetail() {
         }
         queryDetail.setEndTime(endTime);
         queryDetail.setLatency(elapseMs);
-        queryDetail.setWorkGroupName(ctx.getWorkGroup() != null ? ctx.getWorkGroup().getName() : "");
```

Comment:
Why remove this?  why QueryDetail needn't work group info?

Reply:
The workgroup field is not used.